pkg/management/mock_test.go

```diff
@@ -12,3 +12,7 @@ package management
 func ptr(value string) *string {
 	return &value
 }
+
+func boolPtr(value bool) *bool {
```

Comment:
It's been a long time I have worked on Go projects but I can remember these functions are usually named ptr, ptrInt, ptrXXX, ...

Reply:
Good catch